bin/steps/geo-libs

```diff
@@ -22,6 +22,7 @@ source "$BIN_DIR/utils"
 
 # If GDAL exists within requirements, use vendored gdal.
 if [[ "$BUILD_WITH_GEO_LIBRARIES" ]]; then
+  mcount "buildvar.BUILD_WITH_GEO_LIBRARIES"
```

Comment:
Using all caps in `mcount` is odd. Did you mean `buildvar.$BUILD_WITH_GEO_LIBRARIES` maybe?

Reply:
I started doing that for buildvars here: https://github.com/heroku/heroku-buildpack-python/blob/master/bin/steps/pip-install#L28

Is there a standard I can follow? I can update the other buildvars to be internally consistent with it if so

Reply:
No standard, just jumped out in this diff :)